charts/kubewarden-controller/templates/deployment.yaml

```diff
@@ -19,6 +21,7 @@ spec:
         {{- if .Values.telemetry.enabled }}
         "sidecar.opentelemetry.io/inject": "true"
         {{- end }}
+        {{- include "kubewarden-controller.annotations" . | nindent 8 }}
```

Comment:
Maybe it's not useful to annotate to have this `Deployment spec.template.metadata.annotations`, that will get included in the Pods. Afterall, the pods are managed by the deployments and so.

Reply:
removed in https://github.com/kubewarden/helm-charts/pull/178/commits/1a2faf835462d99e0f225c1a51c160357933d731, use the already present `Values.podAnnotations` for controller